components/style/values/computed/mod.rs

```diff
@@ -41,6 +44,13 @@ pub trait ToComputedValue {
 
     #[inline]
     fn to_computed_value(&self, _context: &Context) -> Self::ComputedValue;
+
+    #[inline]
+    /// Convert a computed value to specified value form.
+    ///
+    /// This will be used for recascadeing during animation.
+    /// Such to_specified_valued values should recompute to the same value.
+    fn to_specified_value(computed: &Self::ComputedValue, context: &UncomputeContext) -> Self;
```

Comment:
The specified->computed mapping is many-one; so the reverse trait can't exist. Well, it could exist, but with a trait type parameter, not an associated type.